python/wpt/test.py

```diff
@@ -644,17 +644,17 @@ def setUpModule():
         suppress_force_push=True,
     )
 
-    def setup_mock_repo(repo_name, local_repo):
+    def setup_mock_repo(repo_name, local_repo, main_branch_name: str):
```

Comment:
`default_branch` is used in JSON of expectations.

Reply:
good point, renamed in d0808c849b2c0a2f85551689de43ad06964bec5e